typescript/infra/config/environments/mainnet3/gasPrices.json

```diff
@@ -96,23 +96,23 @@
     "decimals": 9
   },
   "ethereum": {
-    "amount": "13.154482028",
+    "amount": "10.988801871",
```

Comment:
wonder if we should put this up to like 15, there have been some spikes recently?

Reply:
yeah that sgtm